### DISCUSSION TITLE
BootstrapBlazor.BarCode and Huawei P30

### DISCUSSION THREAD

**Discussion by SalvatoreFonti** [dotnetcore/BootstrapBlazor#1199] [Q&A]
Hello, are there any known problems with the BootstrapBlazor.BarCode component and the Phone Huawei P30? 
The P30 runs on EMUI version 12.0.0. Changing the camera has no function, 5 cameras are found, 
but only the image from the front camera is displayed, no matter which one you select.

![grafik](https://user-images.githubusercontent.com/37700136/235934985-e45e65c1-3d00-4896-bedb-d5da6c97bd2b.png)

**Comment by ArgoZhang** [comment]
@SalvatoreFonti have no idea about this. I have never used this phone before. Do you have the same problem on other phones

**Comment by SalvatoreFonti** [comment]
Other than that, I don't know of any problems with other cell phones. It is not relevant to us now. Could have been that you knew about problems.
